service/src/test/java/uk/gov/hmcts/reform/fpl/utils/OrderIssuedNotificationTestHelper.java

```diff
@@ -18,37 +19,41 @@
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
+import java.util.function.Supplier;
 
 import static java.nio.charset.StandardCharsets.ISO_8859_1;
 import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.GeneratedOrderType.BLANK_ORDER;
 import static uk.gov.hmcts.reform.fpl.enums.IssuedOrderType.NOTICE_OF_PLACEMENT_ORDER;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.EMAIL;
 import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.POST;
 import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
 import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.formatCaseUrl;
 
-public class NotifyAdminOrderIssuedTestHelper {
+public class OrderIssuedNotificationTestHelper {
 
-    private NotifyAdminOrderIssuedTestHelper() {
+    private OrderIssuedNotificationTestHelper() {
     }
 
     private static final byte[] PDF = {1, 2, 3, 4, 5};
     private static final String EXAMPLE_COURT = "Family Court";
     private static final String callout = "^Jones, SACCCCCCCC5676576567, hearing " + LocalDateTime.now().plusMonths(3)
         .toLocalDate().format(DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM).localizedBy(Locale.UK));
 
-    public static MapDifference<String, Object> verifyNotificationSentToAdminWhenOrderIssued(
-        ArgumentCaptor<Map<String, Object>> dataCaptor, IssuedOrderType issuedOrderType) {
+    public static void verifyNotificationForOrderIssued(ArgumentCaptor<Map<String, Object>> captor,
```

Comment:
i know it is existing method, but now it becomes really over complicated. My understanding is that the goal is to check if 2 maps are equals, so i think that method name should tell about this.
Secondly, not understand why we need Supplier, i understand that there may be need for lazy evaluation, but here i would just pass object directly. 
My understanding is that JSONObject equals method compares reference and this is not what we want here, and this is a reason why we pass third param. But what about something as simple as this:

    public static void assertEquals(Map<String, Object> actual, Map<String, Object> expected){
        JSONAssert.assertEquals(new JSONObject(actual), new JSONObject(expected), true);
    }:

Reply:
thanks - will refactor to use this method, much better